powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/service/OtpService.java

```diff
@@ -60,6 +59,7 @@ public class OtpService {
     private final OtpRepository otpRepository;
     private final ServiceCatalogue serviceCatalogue;
     private final OtpValueConverter otpValueConverter;
+    private final UserContactConverter userContactConverter = new UserContactConverter(); //TODO: Review autowiring
```

Comment:
What is the problem with autowiring? The converter is not a spring bean and not mentioned in the constructor.

Reply:
The problem is that we use this inconsistently - `OtpValueConverter` is a component and autowired, `UserContactConverter` isn't (elsewhere in the code).